src/components/Common/AvatarEditModal.tsx

```diff
@@ -79,15 +83,49 @@ const AvatarEditModal = ({
   );
   const { t } = useTranslation();
   const [isDragging, setIsDragging] = useState(false);
+  const isLaptopScreen = useBreakpoints({ lg: true, default: false });
+  const [_, setCameraFacingMode] = useState("user");
```

Comment:
```suggestion
  const [ _cameraFacingMode , setCameraFacingMode] = useState("user");
```
should we do this instead ?